torch/_torch_docs.py

```diff
@@ -5278,6 +5278,41 @@ def merge_dicts(*dicts):
             [ 8,  9]])
 """)
 
+add_docstr(torch.nan_to_num,
+           r"""
+nan_to_num(input, nan=0.0, posinf=None, neginf=None, *, out=None) -> Tensor
+
+Replaces :literal:`NaN`, positive infinity, and negative infinity values in :attr:input
+with the values specified by :attr:`nan`, :attr:`posinf`, and :attr:`neginf`, respectively.
+By default, :literal:`NaN`s are replaced with zero, positive infinity is replaced with the
+greatest finite value representable by :attr:`input`'s dtype, and negative infinity
+is replaced with the least finite value representable by :attr:`input`'s dtype.
+
+Args:
+    {input}
+    nan (Number, optional): the value to replace :literal:`NaN`\s with. Default is zero.
```

Comment:
is the "\s" necessary to render it properly?

Reply:
Yeah. Without that CI failing due to warnings. 
Will post the reference stackoverflow link explaining the same if I find it.

Reply:
https://stackoverflow.com/questions/58633196/restructuredtext-inline-code-followed-by-s-results-in-inline-interpreted-text